src/ERC20VotesPartialDelegationUpgradeable.sol

```diff
@@ -30,9 +30,19 @@ abstract contract ERC20VotesPartialDelegationUpgradeable is
    */
   error ERC20ExceededSafeSupply(uint256 increasedSupply, uint256 cap);
 
-  function __ERC20VotesPartialDelegation_init() internal onlyInitializing {}
+  function __ERC20VotesPartialDelegation_init(string calldata _name, string calldata _symbol) internal onlyInitializing {
+    __ERC20_init(_name, _symbol);
+    __EIP712_init(_name, "1");
+    __ERC20Permit_init(_name);
+    __VotesPartialDelegation_init();
+  }
 
-  function __ERC20VotesPartialDelegation_init_unchained() internal onlyInitializing {}
+  function __ERC20VotesPartialDelegation_init_unchained(string calldata _name, string calldata _symbol) internal onlyInitializing {
```

Comment:
We still initialize these even though this contract itself doesn't have state right?

Reply:
nop, updated